<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraConnectDialog</class>
 <widget class="QDialog" name="CameraConnectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>436</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>410</width>
    <height>170</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Connect to Camera</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>14</y>
     <width>411</width>
     <height>408</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_whole">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_top">
      <item>
       <widget class="QGroupBox" name="groupBox_video_source">
        <property name="title">
         <string>选择视频源:</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_usb_camera">
           <item>
            <widget class="QRadioButton" name="usbCameraRadioBtn">
             <property name="font">
              <font>
               <pointsize>9</pointsize>
              </font>
             </property>
             <property name="text">
              <string>USB摄像头设备编号:</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">cameraButtonGroup</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="deviceNumberEdit">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_usb_camera">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QGroupBox" name="groupBox_ip_camera">
             <property name="title">
              <string/>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QFrame" name="frame_ipCamera_top">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <item>
                  <widget class="QRadioButton" name="ipCameraRadioBtn">
                   <property name="enabled">
                    <bool>true</bool>
                   </property>
                   <property name="font">
                    <font>
                     <pointsize>9</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>连接IP摄像机:</string>
                   </property>
                   <property name="checked">
                    <bool>false</bool>
                   </property>
                   <attribute name="buttonGroup">
                    <string notr="true">cameraButtonGroup</string>
                   </attribute>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_ip_camera_address">
                   <property name="text">
                    <string>IP地址:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEdit_ip_camera_address">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>120</width>
                     <height>0</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QFrame" name="frame_ipCamera_bottom">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <item>
                  <widget class="QLabel" name="label_ip_camera_port">
                   <property name="text">
                    <string>端口号:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEdit_ip_camera_port">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>40</width>
                     <height>0</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_ip_camera_user">
                   <property name="text">
                    <string>用户名:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEdit_ip_camera_user">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>50</width>
                     <height>0</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_ip_camera_psw">
                   <property name="text">
                    <string>密码:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEdit_ip_camera_psw">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>80</width>
                     <height>0</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_read_videoStream">
           <item>
            <widget class="QRadioButton" name="readVideoSteamRadioBtn">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="font">
              <font>
               <pointsize>9</pointsize>
              </font>
             </property>
             <property name="text">
              <string>读取网络视频流:</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">cameraButtonGroup</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="videoStreamAddressLineEdit">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>7</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>190</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_read_video_stream">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_img_buffer">
        <property name="font">
         <font>
          <family>Algerian</family>
          <pointsize>9</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>图像缓冲区:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_img_buffer">
        <item>
         <widget class="QLabel" name="label_buffer_size">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>缓冲区大小:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="imageBufferSizeEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_buffer_size_indicator">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>[1-999]</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_img_buffer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="dropFrameCheckBox">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>若视频帧为空则丢弃</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_thread_priority">
        <property name="font">
         <font>
          <family>Algerian</family>
          <pointsize>9</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>线程优先级:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_thread_priority">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_thread_left">
          <item>
           <widget class="QLabel" name="label_capture_thread">
            <property name="font">
             <font>
              <pointsize>9</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>视频捕获线程:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_processing_thread">
            <property name="font">
             <font>
              <pointsize>9</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>图像处理线程:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_thread_right">
          <item>
           <widget class="QComboBox" name="capturePrioComboBox">
            <property name="font">
             <font>
              <pointsize>9</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="processingPrioComboBox">
            <property name="font">
             <font>
              <pointsize>9</pointsize>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_bottom">
      <item>
       <widget class="QPushButton" name="resetToDefaultsPushButton">
        <property name="text">
         <string>恢复默认值</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="okCancelBox">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>419</x>
     <y>1249</y>
     <width>511</width>
     <height>131</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>frame_3</zorder>
  <zorder></zorder>
 </widget>
 <tabstops>
  <tabstop>usbCameraRadioBtn</tabstop>
  <tabstop>deviceNumberEdit</tabstop>
  <tabstop>imageBufferSizeEdit</tabstop>
  <tabstop>dropFrameCheckBox</tabstop>
  <tabstop>capturePrioComboBox</tabstop>
  <tabstop>processingPrioComboBox</tabstop>
  <tabstop>okCancelBox</tabstop>
  <tabstop>resetToDefaultsPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>okCancelBox</sender>
   <signal>accepted()</signal>
   <receiver>CameraConnectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>430</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okCancelBox</sender>
   <signal>rejected()</signal>
   <receiver>CameraConnectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>430</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="cameraButtonGroup"/>
 </buttongroups>
</ui>
